app/components/DataCard.tsx

```diff
@@ -37,7 +37,12 @@ export default function DataCard<T extends Record<string, unknown>>({
   // todo: handling of null/undefined values in select input
   const [editColumn, setEditColumn] = useState<ColDataCardDef<T> | null>(null);
   const { mutate, loading: isMutating } = useMutation(mutation);
-  const [query, typename] = useLookupQuery(editColumn?.lookupTable);
+  const [query, typename] = useLookupQuery(
+    editColumn?.editable && editColumn?.relationship
+      ? editColumn.relationship
+      : undefined
+  );
```

Comment:
we only need to use the lookup query when a column is editable and it has relationship metadata